packages/graphiql/cypress/integration/prettify.spec.js

```diff
@@ -0,0 +1,64 @@
+const prettifiedQuery = `{
+  longDescriptionType {
+    id
+  }
+}
+`;
+
+const prettifiedVariables = `{
+  "a": 1
+}`;
+
+const uglyQuery = `{longDescriptionType {id}}`;
+
+const uglyVaribles = `{"a": 1}`;
+
+const brokenQuery = `longDescriptionType {id}}`;
+
+const brokenVaribles = `"a": 1}`;
+
+describe('GraphiQL Prettify', function() {
+  it('Regular prettification', function() {
+    cy.visit(`/?query=${uglyQuery}&variables=${uglyVaribles}`);
```

Comment:
Please escape the URLs; it doesn't seem to matter for these tests but it'll help future tests get it right :+1: 

```suggestion
    cy.visit(`/?query=${encodeURIComponent(uglyQuery)}&variables=${encodeURIComponent(uglyVaribles)}`);
```